metrics/metricstest/metricsmock.go

```diff
@@ -7,6 +7,11 @@ import (
 	"time"
 )
 
+type MockMetricsOptions struct {
```

Comment:
Do we need mock options? Lets just record all three measurements unconditionally for simplicity.